main/solution/backend/config/infra/cloudformation.yml

```diff
@@ -761,6 +759,11 @@ Resources:
               - sts:AssumeRole
             Resource:
               - !Sub 'arn:aws:iam::${AWS::AccountId}:role/${self:custom.settings.namespace}-prep-raas-master-MasterRole-*'
+          - Effect: Allow
+            Action:
+              - appstream:UpdateImagePermissions
+            Resource:
+              - !Sub 'arn:aws:appstream:${AWS::Region}:${AWS::AccountId}:image/FirefoxPuttyNotepad*'
```

Comment:
This suffix needs to change I suppose to the new ServiceWorkbenchImage name convention.

Reply:
Ahh, good catch, I'll make the update